roles/fixtures/files/deploy_fixtures.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e -u -o pipefail
+
+STATEFILE=~/.current_fixtures
+CONTAINER=quay.io/pulp/pulp-fixtures
```

Comment:
I would probably use docker.io/pulp/pulp-fixtures. We've seen some major outages with quay.io in the past few weeks.